setup.py

```diff
@@ -1,5 +1,11 @@
+from os import path
+
 from setuptools import setup, find_packages
 
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
```

Comment:
Why do you need absolute path here?
```suggestion
with open('README.md', encoding='utf-8') as f:
```

Reply:
Was following official guide (https://packaging.python.org/guides/making-a-pypi-friendly-readme/)